tests/e2e/e2e_test.go

```diff
@@ -70,6 +71,10 @@ var _ = Describe("odoe2e", func() {
 			Expect(config).To(ContainSubstring("true"))
 			Expect(config).To(ContainSubstring("UpdateNotification"))
 		})
+		It("should be checking to see if timeout is the same as the constant", func() {
+			config := runCmd("odo utils config view|grep Timeout")
```

Comment:
you are importing package that is named `config` so this variable conflicts with package name